src/web/app/components/course-edit-form/course-edit-form.component.scss

```diff
@@ -0,0 +1,17 @@
+.invalid-field {
+  padding-top: 5px;
+  color: #B50000;
+}
+
+.form-row {
+  align-items: center;
+}
+
+.fill-plain {
+  background-color: #EAEFF5;
+}
+
+.margin-left-right-1em {
```

Comment:
Seems that this style is not used in `course-edit-form.component.html`, so it can be removed.